python/examples/thrift/server.py

```diff
@@ -35,7 +35,7 @@ def hello(self, name):
 
 def run():
     dispatcher = TornadoDispatcher()
-    TChannel().host(dispatcher).listen(4040)
+    TChannel('localhost:4040').listen(dispatcher)
```

Comment:
See PR description. Port is already known from TChannel construction time. You shouldn't need to specify it again.